pkg/configuration/constants.go

```diff
@@ -22,6 +22,7 @@ const (
 	RAW_CMD_ARGS                    string = "internal_raw_cmd_args"
 	WEB_APP_URL                     string = "internal_snyk_app"
 	INPUT_DIRECTORY                 string = "targetDirectory"
+	UNNAMED_PARAMETER               string = INPUT_DIRECTORY                       //UNNAMED_PARAMETER in the CLI is the positional argument
```

Comment:
It seems weird that the value for this is `"targetDirectory"` when it's not a target directory.

Feels the wrong way around. The value should be `positionalArgument` and the context of that might be a target directory if that's what `positionalArgument` means to a specific command.

Reply:
It might be arbitrary now in the way it works, but can easily create bugs or hard to debug situations when we have to reverse the meaning.

Reply:
Naming is hard and especially in this case. The problem with positional arguments is, that it is something from the cli domain as it describes the concept of arguments that have a meaning based on their position. In a framework that doesn't necessarily know about the application it is used in, the best naming would be usage specific. A mapping in the cli application of the usage based name to the positional argument would be the best. I will adapt the PR accordingly.
Thank you!